src/client/helpers/http.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+
+const getTokens = () => localStorage.getItem('_token') || false;
+
+const http = axios.create({
```

Comment:
Could you use express http or this helper is required?

Reply:
done, renamed to fetchRequest